lax/lichens/sciencerun0.py

```diff
@@ -1,6 +1,8 @@
 # -*- coding: utf-8 -*-
 import numpy as np
+from scipy.stats import binom_test
 from pax import units, configuration
+from pax.InterpolatingMap import InterpolatingMap
```

Comment:
Scipy doesn't have a multidimension map that both interpolates and extrapolates. In principle we could reimplement the pax map here and strip out the logging features, which should keep the extra dependencies to a minimum until the p-value is computed further up the processing chain.